auth.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Pokechat</title>
+    <title>Chat Room</title>
     
     <script src="https://www.gstatic.com/firebasejs/6.1.0/firebase-app.js"></script>
     <script src="https://www.gstatic.com/firebasejs/6.1.0/firebase-auth.js"></script>
```

Comment:
here you need ```<script src="https://www.gstatic.com/firebasejs/6.1.0/firebase-database.js"></script>``` otherwise it throws an error ```firebase.database() is not a function```